### ISSUE TITLE
Flickering (Visual bug)

### ISSUE THREAD

**Issue by Sleep8hoursaDay** [HolographicWings/TOTK-Mods-collection#21]
Hasn't happened in a shrine so far and only in the open world. This bug happens more frequently in snowy sections than grassy ones but happens nonetheless. 
i7-9700K 3.60, 16GB ram, RTX 3070.
![weird bug](https://github.com/HolographicWings/TOTK-Mods-collection/assets/133999104/09901376-96e3-49e2-87a0-7f67510e2001)

![yuzu graphic](https://github.com/HolographicWings/TOTK-Mods-collection/assets/133999104/b033afe1-dd51-4b5a-9e6c-068240e7eb89)
![yuzu advance graphic](https://github.com/HolographicWings/TOTK-Mods-collection/assets/133999104/519ac545-00be-4c40-aa68-25009541f795)

**Comment by akeryuuth** [comment]
It's a bit hard to tell with just one image, could be related to [shadows](https://imgsli.com/MTc5MTM1) or the fact that the resolution is increased threefold. There have been some issues reported with 3x resolution (not 100% sure though).
Does this happen to you if you drop the resolution to 2x ? 
You could also try with the 512x or 1024x Shadows mods to see if anything changes.

**Comment by HolographicWings** [comment]
Someone had the issue in vanilla and adding 512x shadows fixed his issue
https://imgsli.com/MTc5MTM1
You should try the shadows mods in different resolutions
For me that happen for one frame when i zoom and unzoom

**Comment by Sleep8hoursaDay** [comment]
I have tried to put the resolution to 2x (1440p/2160p) which coincidentally or not is also the resolution of my screen and it helped a whole lot. 
So far, it happens once every 5-6 minutes which is good, but makes it hard/long to test with other resolutions.
I've tried the shadow 512x mod, but it doesn't seem to cancel the same almost periodical bug.
I will try the game with other resolutions and will keep you updated if it does NOT occur again.
And thank you very much for the quick responses, it is really appreciated!
